content/2023/04/12/howto-migrate-a-real-life-batch-pipeline-from-the-dataset-api-to-the-datastream-api/index.html

```diff
@@ -0,0 +1,1093 @@
+
+<!DOCTYPE html>
+<html lang="en" dir=>
```

Comment:
Please remove the .html files from the PR. Let the committer generate these once the merge is happening.

Reply:
done